src/context/AppContext.js

```diff
@@ -62,6 +63,8 @@ export default function AppContextProvider({
     // debugger
     const _trace = "AppContext.js/_setBooks()"
     console.log(_trace+":new value for books:", value)
```

Comment:
Could you remove this `console.log` please?

Reply:
Sure, done.